### ISSUE TITLE
Enhanced Visual Feedback, i.e. "Why the HELL did I miss" terminator

### ISSUE THREAD

**Issue by ShibuyaCyana** [2394425147/astrodx#118]
**Is your feature request related to a problem? Please describe.**
It's not unusual to get a miss in a series of "swiped" taps, or at the starting tap of the tap - slide combination. When this happens, player usually think it's AstroDX to blame - and so did I, until I turn on "show touch trailing" in Android developer options...
But it's kinda hard to spot such a thin red trail in game (and it doesn't look good either).

**Describe the solution you'd like**
So, to tell myself why I got a miss:
1. Highlight of active touch indicators, buttons and sensors would dim slowly when finger is released / moving out of sensor zone, rather than instantly;
2. Show a trail of finger movements on screen, which also dims slowly;
3. Spawn judgement text at finger tap position. Spawn a miss (better an alternative one) if it doesn't hit any button / sensor / note.
... and these flashy little VFX might also make this game more visually appealing.

**Describe alternatives you've considered**
Replay system, or simply a screen recorder. lol

**Additional context**
Here's my crappy mock-up:
![image](https://github.com/2394425147/astrodx/assets/39703247/3fb4461b-08c1-4c7a-9dd4-a3e75ca89666)

**Comment by DavidScann** [comment]
When I think about this, this would look extremely distracting to the point where I don't think any normal sensible player would enable this feature. Not only does it occupy the majority of the visual play space, it extends further beyond what most people have set as their play space: Most players I have seen set their play area margin to as low a value as possible.

As a result, I don't think this is a good way to represent user feedback. A replay system is the best system to validate aspects of the game such as this.

**Comment by ShibuyaCyana** [comment]
Yeah I admit the mock-up does look very distracting, cuz I'm a bad at picking colors. It shouldn't be like that. Current beta version do have button state indicators, I just want to keep them lit for a little longer, like these button lights on official cab. 
![image](https://github.com/2394425147/astrodx/assets/39703247/cf359956-92db-4b22-bb78-75004fd3c30c)
Of course, playfield margin could be adjusted like what we have in current release; as for the finger trail, I believe it won't be a problem if put in a lower z-level than the notes.
